cypress/integration/focus-trap-demo.spec.js

```diff
@@ -1,16 +1,20 @@
 describe('focus-trap', () => {
   beforeEach(() => cy.visit('index.html'));
 
-  function verifyCrucialFocusTrapOnClicking(focusedElAlias) {
+  /**
+   * Verify focus trap is **trapping** focus by clicking outside element
+   * @param focusedElSelectorOrAliasInTrap Element in trap that should have focus when outside element is clicked
+   */
+  function verifyCrucialFocusTrapOnClicking(focusedElSelectorOrAliasInTrap) {
```

Comment:
I feel the reworded parameter name expresses itself more close to how it is used